test_data/yast/encryption/autoyast_multi_btrfs.yaml

```diff
@@ -0,0 +1,13 @@
+---
+cryptsetup:
+  device_status:
+    message: is active and is in use.
+    properties:
+      type: LUKS1
+      cipher: aes-xts-plain64
+      device: /dev/vdd1
+      key_location: dm-crypt
+      mode: read/write
+backup_file_info: 'LUKS encrypted file, ver 1 \[aes, xts-plain64, sha256\]'
+cr_test:
+  backup_base_path: '/root/bkp_luks_header_cr_test'
```

Comment:
could please fix documentation [here](https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/10892/files#diff-02e3850fe36faeb4fe731d27cdf08818R187).
And take into account that it should be `backup_path` because now you are pointing to specific file not a base path like before.

Reply:
it sounds right. So i changed the name (in all the corresponding places) and fixed the perlpod in the `lib/validate_encrypt_utils.pm`